docs/notebooks/README.md

```diff
@@ -12,11 +12,14 @@ Lens is primarily a framework for comprehensive assessment of AI models. However
 
 The [Connection](https://github.com/credo-ai/credoai_lens/blob/develop/docs/notebooks/governance_integration.ipynb) notebook walks you through the steps to set up the connection. One connected, there are a lot that can be communicated between the two. In the follow-up [Integration](https://github.com/credo-ai/credoai_lens/blob/develop/docs/notebooks/integration_demo.ipynb) demo, we will take a model created and assessed completely independently of Lens and send that data to Credo AI's Governance Platform.
```

Comment:
Make sure to update `tutorials.rst` as well!

Reply:
Update made